src/macros.jl

```diff
@@ -408,11 +410,24 @@ function parse_one_operator_constraint(_error::Function, vectorized::Bool, sense
     parse_one_operator_constraint(_error, vectorized, Val(:in), aff, set)
 end
 
-function parse_constraint(_error::Function, sense::Symbol, lhs, rhs)
+function parse_constraint(_error::Function, ::Val{:call}, sense::Union{Val{:(<=)}, Val{:(≤)}, Val{:(==)}, Val{:(>=)}, Val{:(≥)}, Val{:(.<=)}, Val{:(.≤)}, Val{:(.==)}, Val{:(.>=)}, Val{:(.≥)}}, lhs, rhs)
     (sense, vectorized) = _check_vectorized(sense)
     vectorized, parse_one_operator_constraint(_error, vectorized, Val(sense), lhs, rhs)...
 end
 
+function parse_constraint(_error::Function, ::Val{:call}, fct::Union{Val{:in}, Val{:∈}}, args...)
+    false, parse_one_operator_constraint(_error, false, Val(:in), args...)...
+end
+
+function parse_constraint(_error::Function, ::Val{:call}, fct::Val, args...)
+    set = function_to_set(_error, fct)
```

Comment:
Move the `function_to_set` function right before this function and give an example in comments